src/array/union/mod.rs

```diff
@@ -296,6 +293,13 @@ impl Array for UnionArray {
         None
     }
 
+    fn is_valid(&self, index: usize) -> bool {
```

Comment:
is this necessary? A union does not have a "validity". I could not find where this was being used.